lib/http-bridge/test/bench/Main.hs

```diff
@@ -199,26 +196,25 @@ parseNetwork = \case
 
 {-# ANN bench_restoration ("HLint: ignore Use camelCase" :: String) #-}
 bench_restoration
-    :: forall (n :: Network) s t key.
+    :: forall (n :: Network) key s t.
         ( IsOwned s key
         , NFData s
         , Show s
         , PersistState s
         , KnownNetwork n
         , t ~ HttpBridge n
         , KeyToAddress t key
+        , PersistKey key
         )
-    => Proxy n
-    -> Proxy key
-    -> (WalletId, WalletName, s)
+    => (WalletId, WalletName, s)
     -> IO ()
-bench_restoration _ _ (wid, wname, s) = withHttpBridge network $ \port -> do
+bench_restoration (wid, wname, s) = withHttpBridge network $ \port -> do
```

Comment:
NOTE: I've removed the proxies in favor of `AllowAmbiguousTypes` + `TypeApplications`. This is equivalent but I find the syntax with type applications much clearer.